### ISSUE TITLE
arrow.nvim like functionality to jump to specific file.

### ISSUE THREAD

**Issue by 231tr0n** [echasnovski/mini.nvim#1134]
### Contributing guidelines

- [X] I have read [CONTRIBUTING.md](https://github.com/echasnovski/mini.nvim/blob/main/CONTRIBUTING.md)
- [X] I have read [CODE_OF_CONDUCT.md](https://github.com/echasnovski/mini.nvim/blob/main/CODE_OF_CONDUCT.md)

### Module(s)

mini.visits

### Description

Hi echasnovski! Is it possible to extend mini.visits to do something like how https://github.com/otavioschwanck/arrow.nvim does it.
Basically we can press ';' key and then navigate to the top few files using numbers. 
I believe this could be achieved using mini.visits altough navigating using numbers is not possible i guess since it uses vim.ui.select.
Is it possible to do something like this or is there a workaround to achieve this.

**Comment by echasnovski** [comment]
There is an exported [`list_paths()`](https://github.com/echasnovski/mini.nvim/blob/608e15779f3d672d30c8ee01ad8ff9cac9f7d723/doc/mini-visits.txt#L542) specifically for users/plugins to programmatically get an array of visited paths in an appropriate order. This seems to be the best common ground for enabling any kind of "select from recent paths" functionality which 'mini.visits' itself does not provide.

Closing as not planned.

**Comment by abeldekat** [comment]
> Basically we can press ';' key and then navigate to the top few files using numbers.
> I believe this could be achieved using mini.visits altough navigating using numbers is not possible i guess since it uses vim.ui.select.

I am not sure, but could this be achieved using the `labeled pickers` discussed [here](https://github.com/echasnovski/mini.nvim/discussions/1109) and `MiniExtra.pickers.visit_paths`?

**Comment by 231tr0n** [comment]
> 
> > Basically we can press ';' key and then navigate to the top few files using numbers.
> > I believe this could be achieved using mini.visits altough navigating using numbers is not possible i guess since it uses vim.ui.select.
> 
> I am not sure, but could this be achieved using the `labeled pickers` discussed [here](https://github.com/echasnovski/mini.nvim/discussions/1109) and `MiniExtra.pickers.visit_paths`?

Yeah it should. Thanks for pointing it out. I will look into your implementation tomorrow.
